src/app/product-list.tsx

```diff
@@ -125,15 +119,21 @@ export default function ProductList() {
   return (
     <SafeAreaView className="flex-1">
       <HeaderLogo />
+      <TouchableOpacity
+        onPress={() => router.back()}
+        className="absolute left-2 top-[67] flex-row items-center rounded-full bg-green-600 p-2 px-4"
+      >
+        <Text className="text-md font-bold text-white">Back</Text>
+      </TouchableOpacity>
       <SearchBar setQuery={setQuery} query={query} />
       <SearchResult query={query} clearQuery={clearQuery} />
-      <View className={`${query === '' ? 'mb-52' : 'mb-80'}`}>
+      <View className={`${query === '' ? 'pb-56' : 'pb-80'}`}>
         <ProductsList
           products={productsToDisplay}
           onEndReached={handleLoadMore}
         />
       </View>
-      <FiltersButton products={productsToDisplay} query={query} />
+      <FiltersButton products={productsToDisplay} />
```

Comment:
was the deletion of query intentional?

Reply:
Yes, because it has no use in the component.

Reply:
You're right! Sorry, I got confused :/